content/source/docs/enterprise/sentinel/import/tfstate.html.md

```diff
@@ -10,103 +10,526 @@ description: |-
+actually required here as an identifier cannot start with number.
+
+Some examples of multi-level access are below:
+
+* To fetch all `aws_instance.foo` resource instances within the root module, you
+  can specify `tfstate.resources.aws_instance.foo`. This would then be indexed
+  off of the resource count index (`0`, `1`, `2`, and so on). Note that as
+  mentioned above, these elements must be accessed using square-bracket map
+  notation (so `["0"]`, `["1"]`, `["2"]`, and so on) instead of dotted notation.
+* To fetch all `aws_instance` resources within the root module, you can specify
+  `tfstate.resources.aws_instance`. This would be indexed off of the names of
+  each resource (`foo`, `bar`, and so on), with each of those maps containing
+  instances indexed on count index as per above.
+* To fetch all resources indiscriminately within the root module,
+  `tfstate.resources` will do. This is then indexed by resource type
+  (`aws_instance`, for example), with the other maps cascading down from there.
+
+Further explanation of the namespace will be in the context of resources. As
+mentioned, when operating on data sources, use the same syntax, except with
+`data` in place of `resources`.
+
+### Value: `attr`
+
+* **Value Type:** A string-keyed map of values.
+
+The `applied` value within the [resource
+namespace](#namespace-resources-data-sources) is a direct mapping to the state
+of the resource. 
+
+The map is a complex representation of these values with data going as far down
+as needed to represent any state values such as maps, lists, and sets.
+
+As an example, given the following resource:
+
+```hcl
+resource "null_resource" "foo" {
+  triggers = {
+    foo = "bar"
+  }
+}
+```
+
+The following policy would evaluate to `true`, if the resource was in the state:
+
+```python
+import "tfstate"
+
+main = rule { tfstate.resources.null_resource.foo[0].attr.triggers.foo is "bar" }
+```
+
+### Value: `depends_on`
+
+* **Value Type:** A list of strings.
+
+The `depends_on` value within the [resource
+namespace](#namespace-resources-data-sources) contains the dependencies for the
+resource.
+
+This is a list of strings which will contain the resource address relative to
+the module.
+
+As an example, given the following resources:
+
+```hcl
+resource "null_resource" "foo" {
+  triggers = {
+    foo = "bar"
+  }
+}
+
+resource "null_resource" "bar" {
+  # ...
+
+  depends_on = [
+    "null_resource.foo",
+  ]
+}
+```
+
+The following policy would evaluate to `true`, if the resource was in the state:
+
+```python
+import "tfstate"
+
+main = rule { tfstate.resources.null_resource.bar[0].depends_on contains "null_resource.foo" }
+```
+
+### Value: `id`
+
+* **Value Type:** String.
+
+The `id` value within the [resource
+namespace](#namespace-resources-data-sources) contains id of the resource.
+
+As an example, given the following data source:
+
+-> Note that we use a _data source_ here because the
```

Comment:
I'd suggest moving this note to somewhere that isn't in the middle of the example flow. Maybe just interchange the line above and this note. And (similar to other comment earlier) use the convention `-> **Note**: We use`